lighthouse-core/test/audits/installable-manifest-test.js

```diff
@@ -35,12 +35,14 @@ describe('PWA: webapp install banner audit', () => {
   describe('basics', () => {
```

Comment:
really a lot of these tests could be dropped now since most of them are just testing are errorIds mapped to strings in the items, but there's not any harm in leaving them in place